vent/controller/control_module.py

```diff
@@ -774,23 +801,28 @@ def __init__(self, pid_control = False, save_logs = True, flush_every = 10, conf
         ControlModuleBase.__init__(self, pid_control, save_logs, flush_every)
         self.HAL = io.Hal(config_file)
         self._sensor_to_COPY()
-    
+
+    def __del__(self):
```

Comment:
i think the order should probably be reversed for code longevity -- assume the base `__del__` method does irreparable cleanup to the object, so any derived ops should come before that.

Reply:
done